cmd/flow/main.go

```diff
@@ -46,8 +44,8 @@ func init() {
 	cmd.AddCommand(blocks.Cmd)
 	cmd.AddCommand(collections.Cmd)
 	cmd.AddCommand(keys.Cmd)
-	cmd.AddCommand(emulator.Cmd)
-	cmd.AddCommand(cadence.Cmd)
+	// cmd.AddCommand(emulator.Cmd)
+	// cmd.AddCommand(cadence.Cmd)
```

Comment:
Why are these commented out?

Reply:
This was back when we hadn't update the sdk so these imports weren't working. Will add back